### ISSUE TITLE
MDN should warn of deprecated aliased theme properties

### ISSUE THREAD

**Issue by wbamberg** [mdn/sprints#821]
# Request type
- [ ] New documentation
- [x] Correction or update

## Page to change
If this request relates to an existing article, please give its URL: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/theme

# Details
Originally filed in Bugzilla: https://bugzilla.mozilla.org/show_bug.cgi?id=1508428

## Related issues
https://bugzilla.mozilla.org/show_bug.cgi?id=1508428

# Acceptance criteria
- [x] Deprecated properties have been marked as deprecated

**Comment by rpl** [comment]
@wbamberg another issue related to this is mozilla/addons-linter#2259, which tracks the "addons-linter" side of the alias deprecation, the related PR (mozilla/addons-linter#2283) is almost ready, the only change still needed on it should be replace the short link that will be part of linter the warning message, with one that points to the section of the MDN page that will contain the additional details on the deprecated aliases.

One option to unblock and land mozilla/addons-linter#2283 in the meantime may be to agree upfront on which section will contain the needed content, so that I can generate the short link right now, even if the content is not there yet.

@wbamberg could https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/theme#Aliases be a good section to point the short link to?

**Comment by wbamberg** [comment]
@rpl , sorry to be slow. Yes, that sounds good to me, but @irenesmith now looks after the WebExtension docs, so we should see what she thinks.

**Comment by jmswisher** [comment]
@irenesmith These notes will need to be updated to change their when Fx69 ships (from "will be" to "are"), or if the target release changes. The BCD will need to be updated at that time, too. Might want to put that in your calendar.
